.github/workflows/ci-java.yml

```diff
@@ -0,0 +1,28 @@
+# https://docs.github.com/en/actions/guides/building-and-testing-java-with-maven
+name: Java CI
+
+on: [ push ]
```

Comment:
Please limit the launch triggers a bit more, so the workflow does not run whenever somebody pushes. You can look in the cpp workflow how to do it:
https://github.com/compose-generator/ccom/blob/main/.github/workflows/ci-cpp.yml#L4-L8

Reply:
nice, that's a good suggestion, will add this